packages/widgets/src/indexer-manager/component.tsx

```diff
@@ -0,0 +1,83 @@
+"use client";
+
+import { useState } from "react";
+import { Anchor, Button, Card, Container, Flex, Group, ScrollArea, Text } from "@mantine/core";
+import { IconCircleCheck, IconCircleX, IconReportSearch, IconTestPipe } from "@tabler/icons-react";
+
+import { clientApi } from "@homarr/api/client";
+import type { Indexer } from "@homarr/integrations/types";
+import { useI18n } from "@homarr/translation/client";
+
+import type { WidgetComponentProps } from "../definition";
+import { NoIntegrationSelectedError } from "../errors";
+
+export default function IndexerManagerWidget({
+  options,
+  integrationIds,
+  serverData,
+}: WidgetComponentProps<"indexerManager">) {
+  if (integrationIds.length === 0) {
+    throw new NoIntegrationSelectedError();
+  }
+  const t = useI18n();
+  const [indexersData, setIndexersData] = useState<{ integrationId: string; indexers: Indexer[] }[]>(
+    serverData?.initialData ?? [],
+  );
+
+  const { mutate: testAll, isPending } = clientApi.widget.indexerManager.testAllIndexers.useMutation();
+
+  clientApi.widget.indexerManager.subscribeIndexersStatus.useSubscription(
+    { integrationIds },
+    {
+      onData() {
+        setIndexersData((data) => {
+          return [...data];
+        });
```

Comment:
Don't you want to take the data from the ``onData(T)`` function?
Right now there will not be any data being updated. Please fix.